resources/build/sentry-control.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+#
+# This script centralises sentry symbol uploads and releases
+# Note: We must install sentry-cli on the build agents
+#
+
+
+## TODO: centralise debug information file (dif) uploads
+
+
+function isSentryConfigured() {
+  if [ -z ${SENTRY_AUTH_TOKEN-} ] || [ -z ${SENTRY_ORG} ] || [ -z ${SENTRY_URL} ]; then
+    echo "WARNING: Sentry environment variables SENTRY_AUTH_TOKEN, SENTRY_ORG and SENTRY_URL must be configured."
+    return 1
+  fi
+  return 0
+}
+
+function isSentryCliAvailable() {
+  which sentry-cli > /dev/null && return 0
+  echo "WARNING: sentry-cli could not be found. Skipping all sentry integration."
+  return 1
+}
+
+function makeSentryRelease() {
+  if isSentryConfigured; then
+    if isSentryCliAvailable; then
+      local SENTRY_RELEASE_VERSION="release-$VERSION_WITH_TAG"
+      echo "Making a Sentry release for tag $SENTRY_RELEASE_VERSION"
+      sentry-cli releases new -p keyman-android -p keyman-developer -p keyman-ios -p keyman-linux -p keyman-mac -p keyman-web -p keyman-windows $SENTRY_RELEASE_VERSION
+      # Testing: sentry-cli releases new -p keyman-windows $SENTRY_RELEASE_VERSION
+
+      echo "Setting commits for release tag $SENTRY_RELEASE_VERSION"
+      sentry-cli releases set-commits --auto $SENTRY_RELEASE_VERSION
```

Comment:
Would you [finalize](https://docs.sentry.io/cli/releases/#finalizing-releases) the release?

Reply:
It's on my agenda. Will consider how I want to do this over the next week or so

Reply:
It's on my agenda. Will consider how I want to do this over the next week or so. In a new PR though.